src/cdcacm_serial.c

```diff
@@ -97,12 +96,13 @@ static void write_data(struct device *dev, const char *buf, int len)
 void cdc_acm_tx()
 {
 	uint32_t dtr = 0;
+	uint8_t write_buffer[BUFFER_LENGTH];
```

Comment:
I reduced the buffer size. 
From my experiments we never get close to actually filling up the buffer because there is a delay on the ARC side